src/ResourceManager/RecoveryServices/Commands.RecoveryServices/AzureRM.RecoveryServices.Netcore.psd1

```diff
@@ -54,7 +54,14 @@ PowerShellVersion = '5.1'
 RequiredModules = @(@{ModuleName = 'AzureRM.Profile.Netcore'; ModuleVersion = '0.12.0'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Management.RecoveryServices.dll', 
+RequiredAssemblies = '.\Microsoft.Azure.Commands.RecoveryServices.ARM.dll', 
```

Comment:
You can remove this from RequiredAssemblies as it is already found in NestedModules: .\Microsoft.Azure.Commands.RecoveryServices.ARM.dll

Edit: I'll do this for you.